otter/worker/launch_server_v1.py

```diff
@@ -325,7 +340,11 @@ def _create_server():
 
         If not, and if no further errors occur, server creation can be retried.
         """
-        d = create_server_sem.run(_create_with_delay)
+        sem = get_sempahore("create_server", "worker.create_server_limit")
```

Comment:
My preference would be to have a semaphore object in a namespace somewhere rather than using strings here.  The general idea is that I always like to find out about missing objects via a `NameError` or `AttributeError`, not a `KeyError` upon looking something up in a dictionary; reason being, if you have something that wishes to share the `"create_server"` semaphore, it shouldn't be able to accidentally and silently get a different semaphore if you have a typo.  But you may feel free to ignore this.